### ISSUE TITLE
Backpack GUI not rendering correctly on Curios Slot

### ISSUE THREAD

**Issue by Kyrnterra** [P3pp3rF1y/SophisticatedBackpacks#227]
Issue:
Opening the backpack while it is in the Back Curio slot now doesn't render the backpack fully and correctly. Backpacks function correctly when held in hand, open while on ground or worn in Chestpiece slot. Was working fine yesterday.

Log:
https://paste.ee/p/I6iTK

1. Equip Backpack to Back Curio slot
2. Click hotkey

Screenshot:
https://prnt.sc/1qixdr3

Versions:
Sophisticated Backpacks: sophisticatedbackpacks-1.16.5-3.2.7.326
Curios API (Forge): curios-forge-1.16.5-4.0.5.2

Restarting seems to have fixed it?

**Comment by P3pp3rF1y** [comment]
This is working just fine for me. That's what I am always testing with anyway. 

But when that happened was that on server? And did you by any chance modify the backpack stack using some tool? Modified number of slots in config? And is it still happening for you or was just one time occurence?

**Comment by kevdevrev** [comment]
I also had this issue today. Never had it before which was strange. Also fixed by restarting game.

**Comment by Abcentic** [comment]
I also had this issue today as well, but the issue still remained when I restarted the game. It only happens when I equip the backpack into the Curios back slot, but its completely fine when I use it in my inventory. I redownloaded the game and that fixed the problem. As soon as I entered LAN with my friend, the issue came back again... so I ended up giving up

Screenshot:
![BackpackGUIissue](https://user-images.githubusercontent.com/90235721/132286460-15f6cc84-f55b-4f3c-b341-10a578339d87.png)
